subsys/mpsl/fem/nrf21540_gpio/mpsl_fem_nrf21540_gpio.c

```diff
@@ -293,7 +293,7 @@ static int mpsl_fem_init(const struct device *dev)
 	return fem_nrf21540_gpio_configure();
 }
 
-SYS_INIT(mpsl_fem_init, POST_KERNEL, CONFIG_KERNEL_INIT_PRIORITY_DEVICE);
+SYS_INIT(mpsl_fem_init, POST_KERNEL, CONFIG_KERNEL_INIT_PRIORITY_DEFAULT);
```

Comment:
I'm afraid if this is done this way at some point we will forget, why the change was introduced. Is there any way to make this initialization priority relative to SDC initialization priority?

Reply:
One way that comes to my mind is that there could be a dedicated Kconfig option representing MPSL FEM initialization priority and another one representing HCI driver initialization priority. We could then give them correct default values and put a build assert to enforce the dependency. It feels to be a bit of an overkill though. What do you think? Do you have other ideas?

Reply:
This would be exactly the approach I think I would choose.